mbuild/packing.py

```diff
@@ -44,11 +46,13 @@ def fill_box(compound, n_compounds, box, overlap=0.2, seed=12345):
     n_compounds : int or list of int
     box : mb.Box
     overlap : float
-
+    density : float
```

Comment:
The expected units for `density` should be documented (i.e. kg/m^3).  We should also have this documented for `overlap` as well.